logicrunner/actorsystem.go

```diff
@@ -0,0 +1,5 @@
+package logicrunner
+
+import "github.com/afiskon/go-actors/actor/system"
+
+var GlobalActorSystem = system.New()
```

Comment:
can we get rid of globallity, please? Top level injectable component?

Reply:
ok, fixed.